### ISSUE TITLE
low accuracy

### ISSUE THREAD

**Issue by mosca-ops** [AlexeyAB/darknet#7238]
the accuracy doesn't increase, why?
I'm using yolov2 tiny
my dataset has 3000 photo with 3 classes

this is my cfg:

> [net]
> batch=64
> subdivisions=8
> width=416
> height=416
> channels=3
> momentum=0.9
> decay=0.0005
> angle=0
> saturation = 1.5
> exposure = 1.5
> hue=.1
> 
> learning_rate=0.001
> burn_in=1000
> max_batches = 20000
> policy=steps
> steps=16000,18000
> scales=.1,.1
> 
> [convolutional]
> batch_normalize=1
> filters=16
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=32
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=64
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=128
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=256
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=2
> 
> [convolutional]
> batch_normalize=1
> filters=512
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> [maxpool]
> size=2
> stride=1
> 
> [convolutional]
> batch_normalize=1
> filters=1024
> size=3
> stride=1
> pad=1
> activation=leaky
> 
> 
> [convolutional]
> batch_normalize=1
> size=3
> stride=1
> pad=1
> filters=512
> activation=leaky
> 
> [convolutional]
> size=1
> stride=1
> pad=1
> filters=40
> activation=linear
> 
> [region]
> anchors =  0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828
> bias_match=1
> classes=3
> coords=4
> num=5
> softmax=1
> jitter=.2
> rescore=0
> 
> object_scale=5
> noobject_scale=1
> class_scale=1
> coord_scale=1
> 
> absolute=1
> thresh = .6
> random=1

and this is my result after 20000 iterations:
![image](https://user-images.githubusercontent.com/62932720/104191720-287d8980-541e-11eb-990f-42fc7d6ef90f.png)
